docs/service_def.md

```diff
@@ -23,7 +23,7 @@ The differences are noted below.
 - `arch`: The hardware architecture of the service implementation in the container image. Valid values are those returned from the GOARCH constant in https://golang.org/pkg/runtime/. The anax agent can be configured to define aliases for these values, see https://github.com/open-horizon/anax/blob/master/test/docker/fs/etc/colonus/anax-combined.config.tmpl for an example. A service is deployed to edge nodes with the same hardware architecture.
 - `sharable`: Can be one of 2 values; `singleton` or `multiple`. Services should be defined as multiple in most cases. The value of this field determines how many instances of the service's containers will be running on a node when the service is deployed more than once to the same node. Use `singleton` when the service is going to be used as a dependency by more than one service, AND those services all run together on a single node, AND the service implementation cannot tolerate multiple instances OR there are not enough resources to support multiple instances.
 - `matchHardware`: Unused
-- `requiredServices`: The list of services on which this service directly depends. A service in this list might have it's own required services. When deploying a service to a node, the full dependency tree is analyzed so that leaf services are started first, working recursively up the tree until the top level service is reached, and is started last. However, just because a service's dependencies are started first, does NOT guarantee that the dependencies are ready to process requests when the parent service is started. Parent services should always be prepared to tolerate unavailable dependent services.
+- `requiredServices`: The list of services on which this service directly depends. A service in this list might have it's own required services. When deploying a service to a node, the full dependency tree is analyzed so that leaf services are started first, working recursively up the tree until the top level service is reached, and is started last. However, just because a service's dependencies are started first, does NOT guarantee that the dependencies are ready to process requests when the parent service is started. Parental services must be prepared to tolerate unavailable dependent services at all times.
```

Comment:
I see what you're trying to do here, but we need to refer to the parent service specifically, not call it parental, which while grammatically correct is not technically correct.  This goes into the documentation, which is important.  Same thing for "should always" instead of "must".  If you wish, just remove "always" but keep "should" instead of "must".

Reply:
> I see what you're trying to do here, but we need to refer to the parent service specifically, not call it parental, which while grammatically correct is not technically correct. This goes into the documentation, which is important. Same thing for "should always" instead of "must". If you wish, just remove "always" but keep "should" instead of "must".

ya, done 👍🏻